langchain/vectorstores/myscale.py

```diff
@@ -0,0 +1,427 @@
+"""Wrapper around MyScale vector database."""
+from __future__ import annotations
+
+import json
+import logging
+from hashlib import sha1
+from threading import Thread
+from typing import Any, Dict, Iterable, List, Optional, Tuple
+
+import tqdm
```

Comment:
i dont think this is a required library, may be good to treat as an optional import (although linting should have caught this, so maybe thats not actually true)

Reply:
Yeah you are right. `tqdm` here is just an optional package for user to monitor the progress of insertion. I will try to remove this.